cmd/mayactl/app/command/volumes_list.go

```diff
@@ -18,13 +18,35 @@ package command
 
 import (
 	"fmt"
+	"html/template"
+	"os"
+	"strings"
+	"text/tabwriter"
 
-	"github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	"github.com/openebs/maya/pkg/client/mapiserver"
 	"github.com/openebs/maya/pkg/util"
+	"github.com/openebs/maya/types/v1"
 	"github.com/spf13/cobra"
 )
 
+// VolumeList struct holds the volume's information like status and volume type.
+type VolumeList struct {
```

Comment:
This does not look correct. The struct should have an array of `Volumes` with some `meta` information like count etc.
The struct is names volumelist but does not contain any array list.